app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+import os
+from typing import Any
+
+
+def move_file(command: str) -> Any:
+    parts = command.split()
+    if len(parts) != 3 or parts[0] != "mv":
+        raise ValueError
```

Comment:
Consider providing a more descriptive error message when raising `ValueError`, such as `raise ValueError('Invalid command format')`, to make debugging easier.